Plugins/Flow.Launcher.Plugin.Explorer/Search/ResultManager.cs

```diff
@@ -266,6 +271,20 @@ internal static Result CreateFileResult(string filePath, Query query, int score
             };
             return result;
         }
+
+        public static bool IsMedia(string extension)
+        {
+            if (string.IsNullOrEmpty(extension))
+            { 
+                return false; 
+            }
+            else
+            {
+                return MediaExtensions.Contains(extension.ToLowerInvariant());
+            }
+        }
+
+        public static readonly string[] MediaExtensions = { ".jpg", ".png", ".avi", ".mkv", ".bmp", ".gif", ".wmv", ".mp3", ".flac", ".mp4" };
```

Comment:
@taooceros @VictoriousRaptor I think this is probably better moved to the plugin project, the idea is that plugin does not need to determine and provide whether the result path is a media or not and let Flow determine it, i.e. flow controls whether to display large preview image.

Let me know what you think, we can update it next round.

Reply:
agree. but plugin should still have control to  whether to use large preview. programs should use small preview. just move this function.